website/server/libs/payments.js

```diff
@@ -109,7 +117,19 @@ api.addSubToGroupUser = async function addSubToGroupUser (member, group) {
   if (member.isSubscribed()) {
     let memberPlan = member.purchased.plan;
     let customerHasCancelledGroupPlan = memberPlan.customerId === this.constants.GROUP_PLAN_CUSTOMER_ID && !member.hasNotCancelled();
-    if (customerIdsToIgnore.indexOf(memberPlan.customerId) !== -1 && !customerHasCancelledGroupPlan) return;
+    let ignorePaymentPlan = paymentMethodsToIgnore.indexOf(memberPlan.paymentMethod) !== -1;
+    let ignoreCustomerId = customerIdsToIgnore.indexOf(memberPlan.customerId) !== -1;
+
+    if (ignorePaymentPlan) {
+      txnEmail(FLAG_REPORT_EMAILS, 'invalid-payment-cancel', [
+        {name: 'USERNAME', content: member.auth.local.username},
+        {name: 'UUID', content: member._id},
+        {name: 'EMAIL', content: member.auth.local.email},
```

Comment:
Could we just do  member.auth.local.email || auth.facebook.emails[0] || auth.facebook.email || auth.google.emails[0] || "NO_EMAIL" ?